src/libraries/System.Net.Http/src/System/Net/Http/Headers/CacheControlHeaderValue.cs

```diff
@@ -404,7 +404,7 @@ public static bool TryParse(string? input, [NotNullWhen(true)] out CacheControlH
                     return 0;
                 }
 
-                nameValueList.Add((NameValueHeaderValue)nameValue);
+                nameValueList.Add((NameValueHeaderValue)nameValue!);
```

Comment:
Might be better to use `Debug.Assert(nameValue is not null);` to suppress this one, since we're trusting the GenericHeaderParser to make this non-null.